zerver/tornado/event_queue.py

```diff
@@ -794,15 +790,11 @@ def receiver_is_off_zulip(user_profile_id: int) -> bool:
 
 def maybe_enqueue_notifications(
     *,
-    user_profile_id: int,
+    user_data: UserMessageNotificationsData,
```

Comment:
We may want to in some follow up standardize on `user_notification_data` as the local variable name for this type.